### ISSUE TITLE
Add phonegap.xml to new projects

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#59]
```
See https://github.com/paulb777/AppLaudApp/blob/master/res/xml/phonegap.xml


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 14 Jan 2012 at 9:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 28 Jan 2012 at 7:00
- Added labels: **Milestone-Release1.2.7**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 2 Feb 2012 at 12:01
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
